sites/default/sqlconf.php

```diff
@@ -1,26 +1,29 @@
 <?php
 //  LibreEHR
```

Comment:
Don't include your sqlconf.php file in any pull requests.  This comes from running your test instance from you developement repository instead of just pulling the code to it.  If you do this, you need to use a gitignore file to ensure it does not get picked up.  This has exposed your sql passphrase.  You might want to snuff this quick.

Reply:
@aethelwulffe How am i supposed to do it.Should i add the sqlconf.php to .gitignore.But how should i revert the pr i sent?Please help me .

Reply:
@RAJAGOPALAN-GANGADHARAN you need to update the git index to ignore the file
use this command
git update-index --assume-unchanged path/to/file

Reply:
@naveen17797 Done should i do a new pr now?

Reply:
@RAJAGOPALAN-GANGADHARAN push the changes to your fork, then check the commits under this Pull Request to ensure that your commit is at the bottom of the list.

Reply:
@Trodrige How do i remove all the commits i have posted so far?
And i tried what you said but it says nothing to commit

Reply:
Do you see the last commit you made?
Do you want to delete previous commits or add some changes to this PR?

Check this to remove commits from a PR: https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-pull-request

I hope I'm understanding your problem. Ping me if you face any issues doing that.

Reply:
@Trodrige I wanted to remove all commits i made to this PR and give one final commit which does everything.That is exactly i needed.Thank you so much for your support guys 😊